apollo-federation/tests/query_plan/build_query_plan_tests/fragment_autogeneration.rs

```diff
@@ -450,8 +450,8 @@ fn works_with_key_chains() {
         Fetch(service: "Subgraph1") {
           {
             t {
-              __typename
               id1
```

Comment:
I'd like to see `__typename` at the beginning, since it's easier to spot when debugging.

Reply:
yeah i'll look into this as we shouldn't be modifying the plan

Reply:
fixed in https://github.com/apollographql/router/pull/6960/commits/00458168bbb07fd1a396059acd18288f8a51476e